changelog/6333.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+If a level 1 XRT file does not specify the heliographic longitude of the spacecraft,
+a silent assumption is made that the spacecraft is at zero Stonyhurst
+heliographic longitude (i.e., the same longitude as Earth) is silently assumed.
```

Comment:
```suggestion
heliographic longitude (i.e., the same longitude as Earth).
```
Heh, I don't know what you guys are doing, but https://github.com/sunpy/sunpy/pull/6328/commits/df075b73b5c1b373423bbf70777b0176546030a1 should not have been committed. =P